src/common/app-event-bus/event-bus.ts

```diff
@@ -8,7 +8,8 @@ import { EventEmitter } from "../event-emitter";
 export interface AppEvent {
   name: string;
   action: string;
-  params?: object;
+  destination?: string;
```

Comment:
could `destination` just be kept in `params`? Or are `params` "url params"?

Reply:
Params is more of "payload" or "metadata"

Reply:
Who is expected to assign `destination`?

Reply:
The author of the event. The main implementation will be in lens-ide and the telemetry extension.

Reply:
You can see it here https://github.com/lensapp/lens-ide/pull/108

Reply:
And we would add the `destination: "Segment"` from the extension side.